### PR TITLE
#2279 country selection with flags & location table columns changed

### PR DESCRIPTION
**Pull request by muammertop**

### CODE REVIEW

File: addons/default/visiosoft/location-module/migrations/2020_11_27_124940_visiosoft.module.location__add_abv_country.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	];
+
+	protected $fields = [
+		'abv' =>  'anomaly.field_type.text',
+	];
+
+	protected $assignments = [
+		'abv',
+	];
+}
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/migrations/2020_11_27_124940_visiosoft.module.location__add_abv_country.php:23]: why do we have 2 of the same migration?

File: addons/default/visiosoft/location-module/resources/js/filterLocation.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ $('.filter-country-btn').on('click', function () {
                 countries = callback;
                 resetValue('country', true, false)
                 $.each(countries, function (index, value) {
-                    $('.filter-location-modal .countries').append(item('country', value.id, value.name));
+                    console.log(value)
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/resources/js/filterLocation.js:15]: log?

File: addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig [hunk 3]
```diff
@@ -119,4 +119,5 @@
 
 <script>
     var null_msg = "{{ trans('visiosoft.module.location::message.null_msg') }}!"
+    var flag_img = "{{ img('visiosoft.module.location::images/flags.png').url|raw }}"
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig:122]: are we using this variable?

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/location-module/migrations/2020_11_27_124940_visiosoft.module.location__add_abv_country.php (hunk 1 above)
Comment (muammertop) [reply addons/default/visiosoft/location-module/migrations/2020_11_27_124940_visiosoft.module.location__add_abv_country.php:23]: I removed the other one

Comment (muammertop) [reply addons/default/visiosoft/location-module/migrations/2020_11_27_124940_visiosoft.module.location__add_abv_country.php:23]: I removed the other one @diashalabi

File: addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig (hunk 3 above)
Comment (muammertop) [reply addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig]: kaldırıldı

File: addons/default/visiosoft/location-module/resources/js/filterLocation.js (hunk 2 above)
Comment (muammertop) [reply addons/default/visiosoft/location-module/resources/js/filterLocation.js]: kaldırıldı

**Review by fatihalp — APPROVED** [review]

File: addons/default/visiosoft/location-module/resources/js/filterLocation.js (hunk 2 above)
Comment (spektra2147) [inline addons/default/visiosoft/location-module/resources/js/filterLocation.js]: remove

File: addons/default/visiosoft/location-module/resources/js/filterLocation.js [hunk 4]
```diff
@@ -198,32 +201,31 @@ function SelectOnClick() {
         var id = $(this).attr('data-id');
 
         if ($(this).attr('data-field') == "country") {
-            $('.filter-location-modal,.filter-location-back,.filter-location-modal .countries').hide();
-            $('input[name="country"]').val(id)
+            // $('.filter-location-modal,.filter-location-back,.filter-location-modal .countries').hide();
```
[the remark this answers is not in the document]
Comment (muammertop) [reply addons/default/visiosoft/location-module/resources/js/filterLocation.js]: removed

File: addons/default/visiosoft/advs-module/src/Adv/AdvModel.php [hunk 5]
```diff
@@ -218,13 +218,11 @@ public function getAdvDetailLinkByModel($object, $type = null)
...
             $id = $object->id;
-            $seo = str_slug($object->name);
-            $seo = str_replace('_', '-', $seo);
+            $seo = str_slug($object->slug, '-');
             return \route('adv_detail_seo', [$seo, $id]);
         }
         $id = $object->getObject()->id;
-        $seo = str_slug($object->getObject()->name);
-        $seo = str_replace('_', '-', $seo);
+        $seo = str_slug($object->getObject()->name, '-');
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/src/Adv/AdvModel.php:225]: $object->getObject()->name can you make it slug

File: addons/default/visiosoft/location-module/migrations/2020_10_13_133916_visiosoft.module.location__add_abv_to_countries.php [hunk 6]
```diff
@@ -6,27 +6,18 @@
...
+		'slug' => 'countries',
+	];
+
+	protected $fields = [
+		'abv' =>  'anomaly.field_type.text',
+	];
+
+	protected $assignments = [
+		'abv',
+	];
```
Comment (diashalabi) [inline addons/default/visiosoft/location-module/migrations/2020_10_13_133916_visiosoft.module.location__add_abv_to_countries.php:22]: why do we have 2 of the same migration?

**Review by diashalabi — CHANGES_REQUESTED** [review]

**Review by diashalabi — APPROVED** [review]

**Pull request merged**